src/explore-education-statistics-common/src/components/Tabs.tsx

```diff
@@ -68,7 +68,10 @@ const Tabs = ({ children, id, onToggle }: Props) => {
                   event.preventDefault();
                   setSelectedTab(index);
                   if (typeof onToggle === 'function') {
-                    onToggle({ id: sectionId, title: props.title });
+                    onToggle({
+                      title: `${props.title}`,
```

Comment:
Why are we adding template strings around `props.title` and `sectionId` in this file?